seed/seed.json

```diff
@@ -830,6 +830,32 @@
                 "platform": ["WINDOWS", "MAC", "LINUX", "ANDROID"]
             }
         },
+        {
+            "name": "ConversionsParametersStudy",
```

Comment:
nit: `ConversionsStudy` as we normally just append `Study` to the feature name

Reply:
iOS does not support field trials at this time. We should fix this ASAP on brave-core .

Reply:
@tmancey i carried the name over from what @moritzhaller had originally on `main`.